Animate list component using ReactTransitionGroup@v1x

benszucs opened the pull request:
Animated the list items [#7](https://github.com/manjeetaR/Task-Manager/issues/7).

manjeetaR commented:
> Animated the list items [#7](https://github.com/manjeetaR/Task-Manager/issues/7).

Impressive @benszucs. If you also check with the hover effect, focus, mouse pointer and such user interactions.

benszucs commented:
Thanks @manjeetaR  I've added some button animations, mouse cursors and fixed the margins on the list items. But I'm struggling to fix the conflicts. Any idea how? Many thanks

manjeetaR commented:
> Thanks @manjeetaR I've added some button animations, mouse cursors and fixed the margins on the list items. But I'm struggling to fix the conflicts. Any idea how? Many thanks

Changes will be made to master if any PR is merged. That is when you pull master and resolve conflicts. Meaning check with your changes and upcoming changes. Merge them both so as to not break any functionality. After that, add/commit/push again.
